railties/lib/rails/welcome_controller.rb

```diff
@@ -3,6 +3,7 @@
 require "rails/application_controller"
 
 class Rails::WelcomeController < Rails::ApplicationController # :nodoc:
+  skip_before_action :verify_authenticity_token
```

Comment:
Maybe use the `skip_forgery_protection` wrapper?
https://github.com/rails/rails/blob/c047e21940a0ff42c0882ec4b28e790fa4260f93/actionpack/lib/action_controller/metal/request_forgery_protection.rb#L161
```suggestion
  skip_forgery_protection
```